modules/installation-aws-user-infra-requirements.adoc

```diff
@@ -27,8 +27,8 @@ You need `AWS::EC2::Instance` objects for the following machines:
 can remove it after your cluster deploys.
 * At least three control plane machines. The control plane machines are not
 governed by a MachineSet.
-* Compute machines. You can create these worker machines manually during the
-cluster installation process.
+* Compute machines. You must create at least two compute, or worker, machines
+during installation. These machines are not governed by a Machinset.
```

Comment:
Machinset > MachineSet

Reply:
Fixed. Are you otherwise ok with this change?

Reply:
I'll assume that you are. Merging.